runtime/src/lib.rs

```diff
@@ -740,6 +740,25 @@ impl pallet_aura::Config for Runtime {
 	type AuthorityId = AuraId;
 }
 
+parameter_types! {
+	pub MaxClassMetadata: u32 = 1024;
+	pub MaxTokenMetadata: u32 = 1024;
+}
+
+impl orml_nft::Config for Runtime {
+	type ClassId = u32;
+	type TokenId = u64;
+	type ClassData = pallet_nft::ClassData<BlockNumber, Self::ClassId>;
+	type TokenData = pallet_nft::TokenData;
+	type MaxClassMetadata = MaxClassMetadata;
+	type MaxTokenMetadata = MaxTokenMetadata;
+}
```

Comment:
What is the reason of adding orml nft into runtime?

Reply:
have the same question.

Reply:
need its configuration?

Reply:
It is a dependency of pallet NFT